engine/api/src/main/scala/pl/touk/nussknacker/engine/api/typed/MapLikePropertyTypeDeterminer.scala

```diff
@@ -0,0 +1,28 @@
+package pl.touk.nussknacker.engine.api.typed
+
+import java.util
+
+import pl.touk.nussknacker.engine.api.typed.typing.{Typed, TypedClass, TypingResult, Unknown}
+
+object MapLikePropertyTypeDeterminer {
+
+  def mapLikePropertyType(typ: TypedClass): Option[TypingResult] = typ match {
```

Comment:
can you add some doc what this method should return? So that when we add another case like avro we know what to do? :)

Reply:
Added